logic/concludable/Concludable.java

```diff
@@ -74,18 +80,18 @@ static IsaConstraint copyConstraint(IsaConstraint isa) {
     }
 
     static ValueConstraint<?> copyConstraint(ValueConstraint<?> value) {
+        //NOTE: isa can never exist on a Value Concludable (or else it would be a Isa Concludable).
```

Comment:
Added this note because it wasn't obvious imho why we weren't copying the isa of a value.